admin/services/school.service.js

```diff
@@ -132,6 +143,36 @@ const schoolService = {
       throw validationError
     }
     await schoolDataService.sqlAddSchool(insertDetails, userId)
+  },
+
+  /**
+   * get list of school audit history
+   * @param {number} schoolId
+   */
+  getSchoolAudits: async function getSchoolAudits (urlSlug) {
+    if (!urlSlug) throw new Error('urlSlug is required')
+    const items = await schoolAuditDataService.getSummary(urlSlug)
+    return items.map(i => {
+      return {
+        id: i.id,
+        createdAt: i.createdAt.format('LLL'),
```

Comment:
This is a non-GDS human-readable format.  It would be better to be consistent with the GDS standard, as well as for internal consistency to use Date Utils `formatFullGdsDate` and add the time on, or add a new method there, unless there is a specific requirement for this format.

Reply:
now uses date service